layouts/partials/primary-download-matrix.hbs

```diff
@@ -48,9 +48,9 @@
       </tr>
 
       <tr>
-        <th>Windows Binary (.exe)</th>
-        <td colspan="3"><a href="https://nodejs.org/dist/{{version.node}}/win-x86/node.exe">32-bit</a></td>
-        <td colspan="3"><a href="https://nodejs.org/dist/{{version.node}}/win-x64/node.exe">64-bit</a></td>
+        <th>Windows Binary (.zip)</th>
+        <td colspan="3"><a href="https://nodejs.org/dist/{{version.node}}/node-{{version.node}}-x86.zip">32-bit</a></td>
```

Comment:
I think the `win` suffix should also be included:

`node-{{version.node}}-win-x86.zip`